src/Cronofy/CronofyOAuthClient.cs

```diff
@@ -124,7 +124,6 @@ public AuthorizationUrlBuilder GetAuthorizationUrlBuilder(string redirectUri)
         /// <remarks>
         /// The delegated scopes read_account, read_events, create_event, and delete_event
         /// are requested by default.
-        /// 
```

Comment:
If this was removed due to StyleCop nagging, if I remember correctly the way to fix it but retain the desired layout is to wrap all but the first paragraph in `<para>` like you would a `<p>` tag, eg:

```
/// <remarks>
/// The delegated scopes read_account, read_events, create_event, and delete_event
/// are requested by default.
/// <para>
/// The Enterprise Connect service_account/accounts/manage and
/// service_account/resources/manage are requested by default.
/// </para>
/// </remarks>
```